src/window.ui

```diff
@@ -791,17 +776,38 @@ along with Video Downloader.  If not, see <http://www.gnu.org/licenses/>.
                   <object class="GtkLabel" id="success_msg_wdg">
```

Comment:
The `id` attribute can be removed because it's not used anywhere (also remove the reference in `window.py`).